__tests__/app.test.js

```diff
@@ -119,3 +119,23 @@ describe("GET /api/articles", () => {
       });
   });
 });
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200:responds with an array of comments", () => {
+    return request(app)
+      .get('/api/articles/1/comments')
+      .expect(200)
+      .then((response) => {
+        const body = response.text;
+        const commentArray = JSON.parse(body)
+        expect(Array.isArray(commentArray)).toBe(true);
+      })
+  });
```

Comment:
You need more in the above test 
- How many comments are there? 
- What do the comments look like? Loop through and assert that each comment has the properties that you expect. 
- You should check that each comment has an article_id of 1